e2e/pages/Dashboard.js

```diff
@@ -1,36 +1,53 @@
 const Page = require('./Page');
-const waitingTime = 30000;
+const Workspaces = require('./Workspaces');
+let CommonActions = require('../utils/CommonActions.js');
 class Dashboard extends Page {
+    constructor() {
+        super();
+        this.createProjectButton = '#create-project-button';
+        this.createWorkspaceButton = '#create-workspace-button';
+        this.accountSelector = '.tc-account-selector';
+        this.workspacesTab = '//span[text()="Workspaces"]';
+        this.workspaceNameTextField = '.tc-form__input';
+        this.createSubmit = '.pvXpn__Button--positive';
+    }
     open() {
         super.open('/dashboard');
     }
     clickCreateProjectButton() {
-        browser.click('#create-project-button');
+        CommonActions.waitAndClick(this.createProjectButton);
     }
     setProjectNameTextField(projectName) {
-        browser.setValue('.tc-form__input', projectName);
+        CommonActions.waitAndSetValue(projectName);
     }
     clickSelectorAccountSelector() {
-        browser.click('.tc-account-selector');
+        CommonActions.waitAndClick(this.accountSelector);
+    }
+    setAccountPath(account) {
+        return `//div[text()= "${account}"]`;
     }
     setAccountItem(account) {
-        browser.waitForVisible('.tc-account-selector__option-list',waitingTime);
-        browser.click(`//div[text()= "${account}"]`);
+        CommonActions.waitAndClick(this.setAccountPath(account));
+        //browser.waitForVisible('.tc-account-selector__option-list',30000);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3ACommentedCode)